src/Hangfire.Mongo.Tests/MongoConnectionFacts.cs

```diff
@@ -14,6 +14,7 @@
 using MongoDB.Bson;
 using MongoDB.Driver;
 using Moq;
+using Newtonsoft.Json;
```

Comment:
Is this reference really needed? You are no longer json (de)serializing but using bson via MongoDB instead.

Same goes for all other using statements ;)